app.js

```diff
@@ -226,12 +235,12 @@ function createServer(app) {
 module.exports = function(options) {
 
     return initApp(options)
-    .then(loadRoutes)
-    .then(function(app) {
-        // serve static files from static/
-        app.use('/static', express.static(__dirname + '/static'));
-        return app;
-    }).then(createServer);
+        .then(loadRoutes)
```

Comment:
We use the code-style where the `.then` starts on the new line without indentation.